wremnants/CardTool.py

```diff
@@ -1132,7 +1132,9 @@ def writeHist(self, h, proc, syst, setZeroStatUnc=False, hnomi=None):
 
         hists = {name: h} # always keep original variation in output file for checks
 
+        s = hist.tag.Slicer()
```

Comment:
Is this used?

Reply:
Good catch, it was used when I was doing the eta folding here before, but now not any longer. Should I keep track of that and edit in the next PR? Just to avoid running CI only for that.  I plan to implement a subparser for the unfolding/theory agnostic.